app/src/main/res/layout/layout_current_list_item.xml

```diff
@@ -41,7 +41,7 @@
 
                 <TextView
                     android:id="@+id/txt_Rank"
-                    android:layout_width="78dp"
+                    android:layout_width="100dp"
```

Comment:
Why the change?

If you're changing this, better to go with a constraint instead of a fixed position.